examples/integrations/itk/SelectROI.ipynb

```diff
@@ -0,0 +1,365 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    print(roi_slices, default_roi_slices)
</pre><p>We could make these two print statements so they are easier to compare.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/itkwidgets/pull/703/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Got it!